spec/integration/data_stream/manifest.spec.yml

```diff
@@ -126,6 +126,14 @@ spec:
               type: string
           required:
           - name
+        data_stream:
+          description: Data stream section of index template
+          type: object
+          additionalProperties: false
+          properties:
+            hidden:
+              description: Makes the data stream hidden
+              type: boolean
```

Comment:
If it is defined here in this key, input packages use this key as a reference to introduce the `elasticsearch_index_template`:

https://github.com/elastic/package-spec/blob/main/spec/input/manifest.spec.yml#L105

But it should not be there for input packages.

Reply:
Do you think it could be an issue if input packages can define `index_pattern.data_stream.hidden`? Is there a good reason to only allow it for integrations but not for input packages?

Reply:
I've just realized that it was going to be applied to both types of packages and I wanted to highlight it.
I don't have any reason to just allow this field in one of them 
@jsoriano do you see any issue on allowing this field in input packages?

Reply:
I don't see any usage of this in input packages, but I guess it won't cause problems neither, let's go on like this.